code/modules/client/preference/preferences_mysql.dm

```diff
@@ -332,23 +349,28 @@
 												hair_red='[r_hair]',
 												hair_green='[g_hair]',
 												hair_blue='[b_hair]',
+												secondary_hair_red='[r_hair_sec]',
+												secondary_hair_green='[g_hair_sec]',
+												secondary_hair_blue='[b_hair_sec]',
 												facial_red='[r_facial]',
 												facial_green='[g_facial]',
 												facial_blue='[b_facial]',
+												secondary_facial_red='[r_facial_sec]',
+												secondary_facial_green='[g_facial_sec]',
+												secondary_facial_blue='[b_facial_sec]',
 												skin_tone='[s_tone]',
 												skin_red='[r_skin]',
 												skin_green='[g_skin]',
 												skin_blue='[b_skin]',
-												markings_red='[r_markings]',
-												markings_green='[g_markings]',
-												markings_blue='[b_markings]',
+												marking_colours='[markingcolourslist]',
```

Comment:
Any time you put text into a query, it needs to be passed through `sql_sanitize_text()`, even if it should never contain characters that would break the query string.